packages/data-stores/README.md

```diff
@@ -13,22 +13,19 @@ import React from 'react';
 import { useSelect } from '@wordpress/data';
 import { Verticals } from '@automattic/data-stores';
 
-const VERTICALS_STORE = Verticals.register()
-
-const VerticalSelect = () => ( {
-	const verticals = useSelect( select =>
-		select( VERTICALS_STORE ).getVerticals()
-    );
-
-    return (
-        <ul>
-            { verticals.map( vertical => (
-                    <li key={ vertical.vertical_id }>{ vertical.vertical_name }</li>
-            ) ) }
-        </ul>
-    );
-} );
-
+const VERTICALS_STORE = Verticals.register();
+
+const VerticalSelect = () => {
+	const verticals = useSelect( ( select ) => select( VERTICALS_STORE ).getVerticals() );
+
+	return (
+		<ul>
+			{ verticals.map( ( vertical ) => (
+				<li key={ vertical.vertical_id }>{ vertical.vertical_name }</li>
+			) ) }
+		</ul>
+	);
+};
```

Comment:
Would you like to add `export` the `VerticalSelect` component or add it to a `render()` method to make ESLint happy?

Reply:
Fixed, thanks!